addons/binding/org.openhab.binding.lametrictime/ESH-INF/thing/countdown-app.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="lametrictime"
+                          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                          xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+                          xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+    <thing-type id="countdownApp" listed="true">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="device" />
+        </supported-bridge-type-refs>
+        
+        <label>LaMetric Time Countdown App</label>
+        <description>Provides control for the countdown app on the LaMetric Time</description>
+        
+        <channels>
+            <channel id="configure" typeId="parameterActionType">
```

Comment:
To provide some value for now, rather than delete this completely I will change it to a duration (in seconds) so it can be a simple Number channel. The only thing the user can't do is set and start the countdown in one action now, but all of the functionality is still there.